docs/ABOUT.md

```diff
@@ -1,14 +1,5 @@
-R is a free, open source, programming language and software environment for statistical computing and graphics developed at Bell Laboratories as a GNU project. It is designed as a true computer language, and is written in C, Fortran and R.
+R is an open source programming language for data analysis and statistical computing. It was created by Ross Ihaka and Robert Gentleman at the University of Auckland, and is an implementation of the S programming language combined with lexical scoping semantics inspired by Scheme. The language is interpreted, dynamically typed, and has a strong foundation in functional programming. This makes it well suited to many of the challenges of data analysis.
```

Comment:
This is so great. I love the emboldened **R**s—I had no idea and it made me grin. I like knowing things like that.